AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -97,6 +97,7 @@ coolblue.be#$#.cookie { display: none !important; }
 coolblue.be#$#body[cookienotificationmodalopened] { overflow: visible !important; padding-right: 0 !important; }
 upjs.sk##.gdprcookie
 ad-doge.com###cookieok_bar
+livejasmin.com##[data-testid="Over18NotificationContainer"]
```

Comment:
What about old rule for cookie?
You have also blocked age verification window.
Try those rules
```
livejasmin.com##div[data-testid="PersonalizedContentPopup"]
livejasmin.com#?#div[data-testid="Over18NotificationContainer"] p > a[href$="/cookies-policy"]:upward(1)
```